src/libs/functions/formatOnboardingRedirection.ts

```diff
@@ -0,0 +1,25 @@
+import CozyClient, {
+  generateWebLink,
+  deconstructRedirectLink
+} from 'cozy-client'
+
+export const formatOnboardingRedirection = (
```

Comment:
Is there a reason why we implement this method here and not in cozy-client like you did for `deconstructRedirectLink`?

Reply:
Not really. But maybe we can keep it here because it is more high level.